core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -141,6 +132,7 @@ private int readInternal(byte[] b, int off, int len, ReadType readType, long pos
     int totalBytesRead = 0;
     long currentPosition = pos;
     long lengthToRead = Math.min(len, mStatus.getLength() - pos);
+    Stopwatch stopwatch = createUnstartedStopwatch();
```

Comment:
```suggestion
    // used in positionedRead, so make stopwatch a local variable rather than class member 
    Stopwatch stopwatch = createUnstartedStopwatch();
```